portal-impl/src/com/liferay/portal/service/persistence/impl/OrganizationFinderImpl.java

```diff
@@ -576,6 +591,94 @@ protected int countByOrganizationId(
 		return 0;
 	}
 
+	protected List<Object> doFindO_U_ByC_P(
+		long companyId, long parentOrganizationId,
+		QueryDefinition<?> queryDefinition) {
+
+		Session session = null;
+
+		try {
+			session = openSession();
+
+			StringBundler sb = new StringBundler(5);
+
+			sb.append(StringPool.OPEN_PARENTHESIS);
+
+			String sql = CustomSQLUtil.get(FIND_BY_C_P);
+
+			sql = StringUtil.replace(
+				sql, "SELECT organizationId",
+				"SELECT organizationId, 0 AS userId, name");
+			sql = StringUtil.replace(
+				sql, "(organizationId > ?) AND", StringPool.BLANK);
```

Comment:
Also, I'd prefer if we can find another way of doing this that does not imply this type of replacements. They are usually error prone because a developer may change the original query in a seemingly neutral way but because the exact string in this code might no longer match a bug is created.

I would prefer even to create a new query if necessary, or else leave a [$PLACEHOLDER$] in the original query to make it clear that it will be replaced later.